internal/locate/region_request.go

```diff
@@ -879,7 +879,11 @@ func (s *RegionRequestSender) getRPCContext(
 	case tikvrpc.TiDB:
 		return &RPCContext{Addr: s.storeAddr}, nil
 	case tikvrpc.TiFlashCompute:
-		rpcCtxs, err := s.regionCache.GetTiFlashComputeRPCContextByConsistentHash(bo, []RegionVerID{regionID})
+		stores, err := s.regionCache.GetTiFlashComputeStores(bo)
+		if err != nil {
+			return nil, err
+		}
+		rpcCtxs, err := s.regionCache.GetTiFlashComputeRPCContextByConsistentHash(bo, []RegionVerID{regionID}, stores)
```

Comment:
How about getting tiflash stores inside the `GetTiFlashComputeRPCContextByConsistentHash` method? In this way, I think we need not to check the store label in `GetTiFlashComputeRPCContextByConsistentHash` anymore :thinking:

Reply:
We need to filter alive store, then call GetTiFlashComputeRPCContextByConsistentHash(), so, put it outside.